app/models/form/sales/pages/person2_age.rb

```diff
@@ -6,9 +6,7 @@ def initialize(id, hsh, subsection)
     @description = ""
     @subsection = subsection
     @depends_on = [
-      { "hholdcount" => 2, "details_known_2" => 1 },
-      { "hholdcount" => 3, "details_known_2" => 1 },
-      { "hholdcount" => 4, "details_known_2" => 1 },
+      { "details_known_2" => 1}
```

Comment:
We don't need household count here because the `details_known_2` question wouldn't get routed to in the first place if the household count was wrong